datamodel_code_generator/parser/base.py

```diff
@@ -700,6 +710,10 @@ def __change_from_import(
                     from_, import_ = full_path = relative(
                         model.module_name, data_type.full_name
                     )
+                    if imports.use_exact:
+                        from_, import_ = exact_import(
+                            from_, import_, data_type.reference.short_name
+                        )
```

Comment:
The repo prefer cover almost lines by the unittest.

Reply:
Hiya @koxudaxi, I'm not entirely sure how to test these lines 😓 I can't find any existing tests for this class method (`Parser.__change_from_import`) so I've got nothing to base it on.

Reply:
@alpoi-x I believe there are tests for executing the whole command. You could just take one of these and test the new option.